modules/nf-core/longstitch/main.nf

```diff
@@ -0,0 +1,55 @@
+process LONGSTITCH {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::longstitch=1.0.5"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/longstitch:1.0.5--hdfd78af_0':
+        'biocontainers/longstitch:1.0.5--hdfd78af_0' }"
+
+    input:
+    tuple val(meta) , path (assembly, stageAs: "assembly.fa")
+    tuple val(meta2), path (reads   , stageAs: "reads.fq.gz")
```

Comment:
Is the `meta2` needed?

Reply:
It can be a different id, most of the time it's the same meta, but I had cases where I was using files with associated to different meta.
I would keep it this way, but we can change it if you prefer.

Reply:
I'm usually wary of these things, but keep it.